### PR TITLE
ENH: pip-installable q2lint

### PR DESCRIPTION
**Pull request by thermokarst**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from setuptools import find_packages, setup
+
+setup(
+    name='q2lint',
+    version='0.0.1',
```
Comment (thermokarst) [inline setup.py:13]: Thoughts on the version string?

Comment (jairideout) [reply setup.py:13]: @ebolyen and I are good with any version string here, :+1:

**Review by jairideout — APPROVED** [review]
This is awesome, thanks @thermokarst! `q2lint++`

I'll hold off on merging until the poncho-time PRs are ready and the repos are unfrozen, otherwise this is GTG!

**Comment by jairideout** [comment]
@thermokarst just wanted to note there's a travis failure with the latest commit

**Comment by thermokarst** [comment]
thanks @jairideout, i left it cooking while i was 🚲ing! looks like it boiled over 😞

**Pull request merged**